### PR TITLE
Do not initialize asynchronous notifications if they are disabled

### PR DESCRIPTION
**Pull request by skateman**
Parent PR in the backend: https://github.com/ManageIQ/manageiq/pull/13898
Same PR for the classic UI: https://github.com/ManageIQ/manageiq/pull/13898

https://www.pivotaltracker.com/story/show/140478617

### CODE REVIEW

File: client/app/core/event-notifications.service.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /** @ngInject */
-export function EventNotificationsFactory($timeout, lodash, CollectionsApi, Session, $log, ActionCable) {
+export function EventNotificationsFactory($timeout, lodash, CollectionsApi, Session, $log, ActionCable, ServerInfo) {
   var state = {};
   var toastDelay = 8 * 1000;
```
Comment (chriskacerguis) [inline client/app/core/event-notifications.service.js]: IS this needed since we removed toast?  (not sure if it is used for something else, but should probably be renamed to reflect that if that is the case).

Comment (AllenBW) [inline client/app/core/event-notifications.service.js]: not necessary anymore, (i forgot to kill it sorries :( )

Comment (AllenBW) [inline client/app/core/event-notifications.service.js]: will be done in another pr

**Comment by AllenBW** [comment]
@skateman any inside track on when that pending pr might find its way in?

**Comment by skateman** [comment]
@AllenBW [soon](https://github.com/ManageIQ/manageiq/pull/13898#issuecomment-285794130)

**Comment by AllenBW** [comment]
bahahaha 👍 <3 🙇‍♀️

**Comment by Fryguy** [comment]
#13898 merged

**Comment by chriskacerguis** [comment]
@skateman could you please provide an update on when this can be merged?

**Comment by skateman** [comment]
@chriskacerguis as @Fryguy [merged the core PR](https://github.com/ManageIQ/manageiq-ui-service/pull/547#issuecomment-286195315) maybe now :wink:

**Comment by chriskacerguis** [comment]
@skateman if this is good to go, can you please remove the WIP?
